agr_java_core/src/main/java/org/alliancegenome/neo4j/repository/AlleleRepository.java

```diff
@@ -21,6 +21,7 @@ public Allele getAllele(String primaryKey) {
         query += " MATCH p1=(aSpecies:Species)-[:FROM_SPECIES]-(a:Allele) WHERE a.primaryKey = {primaryKey}";
         query += " OPTIONAL MATCH p3=(a:Allele)-[:IS_ALLELE_OF]-(gene:Gene)-[:FROM_SPECIES]-(gSpecies:Species)";
         query += " OPTIONAL MATCH p4=(a:Allele)-[:ALSO_KNOWN_AS]-(:Synonym)";
+        query += " OPTIONAL MATCH vari=(a:Allele)<-[:VARIATION]-(variant:Variant)--(soTerm:SOTerm)";
```

Comment:
Relationship between variant and soTerm?

Reply:
Adding in the next commit.